packages/liveblocks-client/src/client.node.test.ts

```diff
@@ -31,6 +31,55 @@ function createClientAndEnter(options: ClientOptions) {
 }
 
 describe("createClient", () => {
+  test.each([
+    // [publicApiKey, authEndpoint, errorMessage]
+    [
+      undefined,
+      undefined,
+      "Invalid Liveblocks client options. For more information: https://liveblocks.io/docs/api-reference/liveblocks-client#createClient",
+    ],
+    [
+      null,
+      undefined,
+      "Invalid Liveblocks client options. For more information: https://liveblocks.io/docs/api-reference/liveblocks-client#createClient",
+    ],
+    [
+      undefined,
+      null,
+      "authEndpoint must be a string or a function. For more information: https://liveblocks.io/docs/api-reference/liveblocks-client#createClientAuthEndpoint",
+    ],
+    [
+      "sk_xxx",
+      undefined,
+      "Invalid key. You are using the secret key which is not supported. Please use the public key instead. For more information: https://liveblocks.io/docs/api-reference/liveblocks-client#createClientPublicKey",
```

Comment:
I would replace `key` by `publicApiKey` to give instant feedback on which option is used incorrectly.

```suggestion
      "Invalid publicApiKey. You are using the secret key which is not supported. Please use the public key instead. For more information: https://liveblocks.io/docs/api-reference/liveblocks-client#createClientPublicKey",
```

Reply:
Good improvement! Will adjust